addons/misc/functions/fnc_suitcaseNukeZeus.sqf

```diff
@@ -0,0 +1,211 @@
+
+            _isArmed = _device getVariable [QGVAR(suitcaseNuke_isArmed), false];
+
+            sleep 0.3; 
+        };
+
+        if(_isArmed) then {
+            _device setDamage 1;
+        };
+
+        while {not isNull _device} do {
+            // if(_countdownRemaining > 0) then {
+                { 
+                    _device setObjectTextureGlobal[_x, 
+                        format ['#(rgb,512,512,3)text(0,1,"LCD14",0.3,"#171717","#eb4034","\n%1")', _countdownRemainingText]
+                    ]; 
+                } forEach [1,2,3];
+            // } else {
+            //     // TODO: make this customisable, or remove
+            //     { 
+            //         _device setObjectTextureGlobal[_x, 
+            //             format ['#(rgb,512,512,3)text(0,1,"LucidaConsoleB",0.1,"#171717","#eb4034","\n\n\nGoodbye!!!")']
+            //         ]; 
+            //     } forEach [1,2,3];
+            // };
+            sleep 0.8;
+            { 
+                _device setObjectTextureGlobal[_x, 
+                    '#(rgb,512,512,3)text(0,1,"LCD14",0.3,"#171717","#eb4034","")'
+                ]; 
+            } forEach [1,2,3];
+            sleep 0.4;
+        };
+    };
+
+    // Activation Code
+    _device setVariable [QGVAR(suitcaseNuke_activate), _effect];
+    _device setVariable [QGVAR(suitcaseNuke_code), _customCode];
+    _device setVariable [QGVAR(suitcaseNuke_codeTarget), _target];
+    _device addMPEventHandler ["MPKilled", {
+        params ["_device"];
+        if(isServer) then {
+            switch(_device getVariable QGVAR(suitcaseNuke_activate)) do {
+                case "explosive_small": {
+                    private _explosive = createVehicle ["DemoCharge_Remote_Ammo_Scripted", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    _explosive setDamage 1;
+                    deleteVehicle _device;
+                };
+                case "explosive_large": {
+                    private _explosive = createVehicle ["Bo_GBU12_LGB", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    _explosive setDamage 1;
+                    deleteVehicle _device;
+                };
+                case "explosive_nuke": {
+                    ["zen_modules_moduleNuke", [ASLToAGL(getPosASL _unit), 200, 300, false]] call CBA_fnc_globalEvent;
+                    deleteVehicle _device;
+                };
+                case "smoke_yellow": {
+                    private _smoke = createVehicle ["SmokeShellYellow_Infinite", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    hideObjectGlobal _smoke;
+                    _device setVariable [QGVAR(suitcaseNuke_smoke), _smoke];
+                    _smoke attachTo [_device];
+                    _device addEventHandler ["Deleted", {
+                        params ["_device"];
+                        deleteVehicle (_device getVariable [QGVAR(suitcaseNuke_smoke), objNull]);
+                    }];
+                };
```

Comment:
Almost tempted to suggest checking if EWAR is loaded and adding option for EMP. :P 
It's just an CBA Event to fire an EMP without spawning a "The Device"

Reply:
Added that as a TODO yesterday :D